xeda/xeda_app.py

```diff
@@ -56,64 +59,84 @@ def __init__(self):
 
     def main(self, args = None):
         parsed_args = self.parse_args(args)
-
-        if parsed_args.debug:
-            logger.setLevel(logging.DEBUG)
-
-        runner_cls = parsed_args.flow_runner
-
         toml_path = parsed_args.xedaproject if parsed_args.xedaproject else Path.cwd() / 'xedaproject.toml'
-        xeda_project = {}
-        try:
-            with open(toml_path) as f:
-                xeda_project = sanitize_toml(toml.load(f))
-
-        except FileNotFoundError as e:
-            print(f'Cannot open project file: {toml_path}. Please specify the correct path using --xedaproject', e)
-            exit(1)
-        except IsADirectoryError as e:
-            self.fatal(f'The specified design json is not a regular file.', e)
-            raise e
-
-        if parsed_args.xeda_run_dir is None:
-            rundir = None
-            project = xeda_project.get('project')
-            if isinstance(project, list):
-                project = project[0]
-            if project:
-                rundir = project.get('xeda_run_dir')
-            if not rundir:
-                rundir = os.environ.get('xeda_run_dir')
-            if not rundir:
-                rundir = 'xeda_run'
-            parsed_args.xeda_run_dir = rundir
-
-        xeda_run_dir = Path(parsed_args.xeda_run_dir).resolve()
-        xeda_run_dir.mkdir(exist_ok=True, parents=True)
-
-        logdir = xeda_run_dir / 'Logs'
-        logdir.mkdir(exist_ok=True, parents=True)
-
-        timestamp = datetime.now().strftime("%Y-%m-%d-%H%M%S%f")[:-3]
-        logFormatter = logging.Formatter("%(asctime)s [%(threadName)-12.12s] [%(levelname)-5.5s]  %(message)s")
-
-        logfile = logdir / f"xeda_{timestamp}.log"
-        print(f"Logging to {logfile}")
-
-        fileHandler = logging.FileHandler(logfile)
-        fileHandler.setFormatter(logFormatter)
-        logger.addHandler(fileHandler)
-
-        coloredlogs.install(
-            'INFO', fmt='%(asctime)s %(levelname)s %(message)s', logger=logger)
-
-        logger.info(f"Running using FlowRunner: {runner_cls.__name__}")
-
-        xeda_project['xeda_version'] = __version__
-
-        runner = runner_cls(parsed_args, xeda_project, timestamp)
-
-        runner.launch()
+        if parsed_args.flow == "init":
+            if Path(toml_path).is_file():
+                logger.critical(f"{toml_path} already exists! Exiting")
+                return
+            else:
+                xedaprj_header = {
```

Comment:
I think the new project template should go in a separate file and not inlined

Reply:
Sure, will move it to the templates section